src/sql/src/plan/statement/ddl.rs

```diff
@@ -1027,6 +1023,22 @@ pub fn plan_create_source(
     })
 }
 
+/// Take the inline schemas that have been purified out
+///
+/// Panics if purify doesn't correclty inline schemas
```

Comment:
I think this comment could use another pass. "Take the inline schemas that have been purified out" is IMO both more verbose and less clear than the name of the function ("extract inline schemas"). Also "s/correclty/correctly/" .